crates/bevy_gltf/src/lib.rs

```diff
@@ -121,18 +125,86 @@ pub struct GltfNode {
     pub extras: Option<GltfExtras>,
 }
 
+impl GltfNode {
+    /// Create a node extracting name and index from glTF def
+    pub fn new(
+        node: &gltf::Node,
+        children: Vec<GltfNode>,
+        mesh: Option<Handle<GltfMesh>>,
+        transform: bevy_transform::prelude::Transform,
+        extras: Option<GltfExtras>,
+    ) -> Self {
+        Self {
+            index: node.index(),
+            name: node.name().map(|s| s.to_string()),
+            children,
+            mesh,
+            transform,
+            extras,
+        }
+    }
+
+    /// Computed name for a node - either a user defined name or a generated name from index
+    pub fn name(&self) -> String {
+        if let Some(name) = &self.name {
+            name.clone()
+        } else {
+            format!("GltfNode{}", self.index)
+        }
+    }
+
+    /// Computed asset label for node based on its index in the scene
+    pub fn label(&self) -> String {
+        format!("Node{}", self.index)
```

Comment:
could you comment on your choice to not go for the same string in both those cases?

Reply:
This is the asset label within GLTF asset - I will rename for clarity.